client/src/main/java/com/paypal/selion/platform/grid/MobileNodeType.java

```diff
@@ -55,6 +57,12 @@ public static String getSupportedMobileNodesAsCSV() {
      * @return - A {@link MobileNodeType} enum that represents a SeLion compliant mobile node type.
      */
     public static MobileNodeType getMobileNodeType(String mobileNodeType) {
+        // this is a mandatory field ??? should we default it to appium if they haven't specified since appium supports
+        // both ios and android
+        if (StringUtils.isEmpty(mobileNodeType)) {
+            
```

Comment:
Sloppy me.  We had a discussion where should we force the user to enter in a mobileNodeType(current implementation), or default it to appium for them if they dont explicitly define a mobileNodeType.  I was in thought process here and forgot to take this out.  I will take it out and address the mandatory-ness in a different change.